tests/core/test_grad.py

```diff
@@ -27,9 +27,15 @@ def _test_audio_grad(attr: str, target=True, kwargs: dict = {}):
                 # If necessary, propagate spectrogram changes to waveform
                 if result.stft_data is not None:
                     result.istft()
-                result.audio_data.sum().backward()
+                if result.audio_data.dtype.is_complex:
+                    result.real().sum().backward()
```

Comment:
Should this be `result.audio_data.real().sum().backward()`?

Reply:
Yeah that seems right, will fix